desktop/notification/gtk.go

```diff
@@ -0,0 +1,100 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package notification
+
+import (
+	"time"
+
+	"github.com/godbus/dbus"
+)
+
+const (
+	gtkBusName    = "org.gtk.Notifications"
+	gtkObjectPath = "/org/gtk/Notifications"
+	gtkInterface  = "org.gtk.Notifications"
+)
+
+type gtkBackend struct {
+	conn      *dbus.Conn
+	manager   dbus.BusObject
+	desktopID string
+	firstUse  time.Time
+}
+
+// TODO: support actions via session agent.
+var newGtkBackend = func(conn *dbus.Conn, desktopID string) (NotificationManager, error) {
+	// If the D-Bus service is not already running, assume it is
+	// not available.
+	var owner string
```

Comment:
is owner used for anything ?

Reply:
thanks for the explanation, @stolowski probably worth a comment in the code that we just use owner here to ensure that the method call works with the expected types